Containerfile.bpfman.multi.arch

```diff
@@ -1,53 +1,40 @@
 # We do not use --platform feature to auto fill this ARG because of incompatibility between podman and docker
 ARG BUILDPLATFORM=linux/amd64
 
-FROM --platform=$BUILDPLATFORM rust:1 AS bpfman-build
+FROM --platform=$BUILDPLATFORM redhat/ubi9-minimal AS bpfman-build
```

Comment:
I see u changed the base image but I don't see where u install rust/cargo and openssl pkg ?

Reply:
For this, binaries are built on the host not in the container. So rust is not needed in the container.